bitnami/kubeapps/Chart.lock

```diff
@@ -1,12 +1,12 @@
 dependencies:
 - name: redis
   repository: https://charts.bitnami.com/bitnami
-  version: 16.13.1
+  version: 17.0.1
 - name: postgresql
   repository: https://charts.bitnami.com/bitnami
-  version: 11.6.12
+  version: 11.6.16
```

Comment:
You should notice in the PR the rest of the upgrades...

Reply:
Those are automatically updated to the last version available matching 11.x.x (at Chart.yaml) when running 'helm dep update'.

It is expected that dependencies are regularly updated during releases, but for majors it requires manual action, for example, changing from 16.x.x to 17.x.x the target version as in this case.

Reply:
It would be great to notice it in the PR description.